startup/15-electrometer.py

```diff
@@ -1,8 +1,13 @@
 from ophyd import Signal, EpicsSignalWithRBV, Component as Cpt
-from ophyd.areadetector import ADBase
-from ophyd.quadem import QuadEM 
+from ophyd.areadetector import ADBase, ADComponent as ADCpt
+from ophyd.quadem import QuadEM
 from ophyd import DynamicDeviceComponent as DDCpt
 from collections import OrderedDict
+from nslsii.ad33 import QuadEMV33, StatsPluginV33
```

Comment:
We don't use the imported `StatsPluginV33` anywhere (as we originally wanted), so I am suggesting removing it.
```suggestion
from nslsii.ad33 import QuadEMV33
```